cmd/root.go

```diff
@@ -37,6 +38,28 @@ func mustFlagString(cmd *cobra.Command, name string, required bool) string {
 	return val
 }
 
+type EnrollTokenData struct {
```

Comment:
we should use viper instead. viper is built by the same people that built cobra and it makes it trivial to create configs that also bind to command line arguments and environment variables.

https://github.com/spf13/viper

also we should probably choose something a bit more user friendly in the config format like TOML